cli/lib/cert-lib.js

```diff
@@ -356,7 +367,9 @@ function createVault(username, password, managementUri, organization, environmen
 
 function addKeyToVault(username, password, managementUri, organization, environment, vaultName, key, value, cb) {
 
-  const uri = util.format('%s/v1/organizations/%s/environments/%s/vaults/%s/entries', managementUri, organization, environment, vaultName);
+  //const uri = util.format('%s/v1/organizations/%s/environments/%s/vaults/%s/entries', managementUri, organization, environment, vaultName);
```

Comment:
Is it desirable to keep this stuff in here, but commented out? May just confuse us down the line.